crates/publish/src/lib.rs

```diff
@@ -4,11 +4,14 @@
 
 mod bindle_pusher;
 mod bindle_writer;
+mod error;
 mod expander;
 
 pub use bindle_pusher::push_all;
 pub use bindle_writer::write;
+pub use error::{Error, PublishResult};
 pub use expander::expand_manifest;
+pub use expander::prepare_bindle;
```

Comment:
`expand_manifest` and `write` are no longer used anywhere. Does it make sense to make them private?

Reply:
If they're fully superseded by `prepare_bindle` then yeah, may as well

Reply:
Extracted it into https://github.com/fermyon/spin/pull/887